library/classes/Installer.class.php

```diff
@@ -757,7 +776,7 @@ public function install_gacl()
         $gacl->add_acl(
             array(
                 'acct' => array('bill', 'disc', 'eob', 'rep', 'rep_a'),
-                'admin' => array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','menu','manage_modules'),
+                'admin' => array('calendar', 'database', 'forms', 'practice', 'superbill', 'users', 'batchcom', 'language', 'super', 'drugs', 'acl','multipledb','menu','manage_modules','drugs'),
                 'encounters' => array('auth_a', 'auth', 'coding_a', 'coding', 'notes_a', 'notes', 'date_a', 'relaxed'),
```

Comment:
Also should add the new acos to the Administration and Emergency acls

Reply:
OK.